src/app/layout/layout.module.ts

```diff
@@ -0,0 +1,36 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatToolbarModule } from '@angular/material/toolbar';
+import { MatIconModule } from '@angular/material/icon';
+import { MatSidenavModule } from '@angular/material/sidenav';
+import { MatListModule } from '@angular/material/list';
+import { TranslateModule } from '@ngx-translate/core';
+import { AvatarModule} from 'ngx-avatar';
+import { LayoutRoutingModule } from './layout-routing.module';
+import { UserInfoComponent } from './components/user-info/user-info.component';
+import { AppbarComponent } from './components/appbar/appbar.component';
+import { NavItemComponent } from './components/nav-item/nav-item.component';
+import { LayoutComponent } from './components/layout/layout.component';
+
+@NgModule({
+  declarations: [
+    LayoutComponent,
+    UserInfoComponent,
+    AppbarComponent,
+    NavItemComponent
+  ],
+  imports: [
+    CommonModule,
+    MatToolbarModule,
+    MatIconModule,
+    MatSidenavModule,
+    MatListModule,
+    TranslateModule,
+    AvatarModule,
+    LayoutRoutingModule
+  ],
+  exports: [
+    LayoutComponent
```

Comment:
why is LayoutComponent exported here? as far as I can see it is only used inside of LayoutModule so it it not necessary.